### PR TITLE
Fits with non-negative brightness

### PR DESCRIPTION
**Pull request by rbooth200**
I've added the option to produce non-negative fits. 

The code proceeds as normal, but right at the end I am finding the best-fit non-negative brightness given the MAP power spectrum. I've added an option to the bootstrap to use this instead of the mean, if desired.

### CODE REVIEW

File: frank/fit.py [hunk 1]
```diff
@@ -21,6 +21,7 @@
    and output results. Alternatively a custom parameter file can be provided.
 """
 
+from frank import io, geometry, make_figs, radial_fitters, utilities
```
Comment (jeffjennings) [inline frank/fit.py]: Sorry can you change this back? Marco specifically asked that all frank imports go after standard python imports.

Comment (rbooth200) [reply frank/fit.py]: Sure

**Comment by jeffjennings** [comment]
Sorry I see that by leaving a comment I opened a review; do you want me to review now or wait?

**Comment by rbooth200** [comment]
Yes review now

File: frank/parameter_descriptions.json [hunk 2]
```diff
@@ -46,6 +46,8 @@
   },
 
   "analysis" : {
-    "bootstrap_ntrials": "Number of trials (dataset realizations) to perform in a bootstrap analysis"
+    "bootstrap_ntrials": "Number of trials (dataset realizations) to perform in a bootstrap analysis",
+    "bootstrap_non_negative" : "Determines whether the mean (if false) or best-fit non-negative solution in used in the bootstrap"
```
Comment (jeffjennings) [inline frank/parameter_descriptions.json]: Very minor, but can you update to 

"bootstrap_non_negative" : "Whether the mean (if false) or best fit nonnegative solution in used in the bootstrap"

Comment (rbooth200) [reply frank/parameter_descriptions.json]: Done

File: frank/radial_fitters.py [hunk 3]
```diff
@@ -745,7 +757,10 @@ def fit(self, u, v, V, weights=1, verbose=False):
                 self._iteration_diagnostics['mean'].append(fit.mean)
 
             count += 1
-
+            
+        if verbose:
+            print()
```
Comment (jeffjennings) [inline frank/radial_fitters.py]: I don't need this to print the line at 732, do you?

Comment (rbooth200) [reply frank/radial_fitters.py]: Its there so add a new line after the "    FrankFitter iteration XXX" for the last iteration

**Comment by rbooth200** [comment]
See comments

File: frank/radial_fitters.py (hunk 3 above)
Comment (jeffjennings) [reply frank/radial_fitters.py]: The print statement uses flush so that it reprints on the same line at each iteration.

Comment (rbooth200) [reply frank/radial_fitters.py]: Yes, but the line following it will begin on the same line as the iteration statement unless you add that print

Comment (jeffjennings) [reply frank/radial_fitters.py]: ok

**Pull request merged**